crypto/trace.c

```diff
@@ -146,19 +146,32 @@ const char *OSSL_trace_get_category_name(int num)
 {
     size_t i;
 
+    /*
+     * Partial check that OSSL_TRACE_CATEGORY_... macros
+     * are synced with trace_categories array
+     */
+    static const size_t trace_categories_len = OSSL_NELEM(trace_categories);
+    if (!ossl_assert(OSSL_TRACE_CATEGORY_NUM == trace_categories_len))
```

Comment:
Please add empty line above (and probably drop the one after the declaration of i).